### ISSUE TITLE
Explanations of subject_hash_old and issuer_hash_old are reversed.

### ISSUE THREAD

**Issue by taketakeyyy** [openssl/openssl#12382]
# Environment
Windows 10
OpenSSL 1.1.1g  21 Apr 2020

# Bug
```
OpenSSL> x509 -help
Usage: x509 [options]
Valid options are:
 -help                      Display this summary
 ...
 -subject_hash_old          Print old-style (MD5) issuer hash value
 -issuer_hash_old           Print old-style (MD5) subject hash value
```

I think that -subject_hash_old is "Print old-style (MD5) subject hash value"

**Comment by beldmit** [comment]
Would you like to submit a PR?

**Comment by taketakeyyy** [comment]
No, I will not. thanks.

**Comment by beldmit** [comment]
Fixed in #12482 for master and #12505 for 1.1.1
